src/include/network/routines.rb

```diff
@@ -690,10 +690,10 @@ def filter_out(device_info, driver)
 
       # filter out device with chelsio Driver and no Device File or which cannot
       # networking (bnc#711432)
-      if driver == "cxgb4" &&
-          (device_info["dev_name"] || "") == "" ||
-          device_info["vendor_id"] == 70_693 &&
-              device_info["device_id"] == 82_178
+      if (driver == "cxgb4" &&
+          (device_info["dev_name"] || "") == "") ||
```

Comment:
np: I know it is out of the scope, but this condition looks tricky. What about `device_info["dev_name"].to_s.empty?`?